spec/factories/case_groups.rb

```diff
@@ -2,12 +2,17 @@
   factory :case_group do
     transient do
       case_count { 1 }
+      casa_cases { [] }
     end
     casa_org { CasaOrg.first || create(:casa_org) }
     sequence(:name) { |n| "Family #{n}" }
 
     after(:build) do |case_group, evaluator|
-      if case_group.case_group_memberships.empty?
+      if evaluator.casa_cases.size > 0
+        evaluator.casa_cases.each do |casa_case|
+          case_group.case_group_memberships.build(casa_case: casa_case)
+        end
+      elsif case_group.case_group_memberships.empty?
```

Comment:
This feels excessively complex but I can't think of a better approach at this moment.

Reply:
hmm... nice eye. I think I could set casa_cases, count with ORs and scrap the if/else maybe, will try it out.

Reply:
a little better, but didn't want to break any other specs.